kuttl-test.yaml

```diff
@@ -0,0 +1,25 @@
+#
+# EXECUTION (from install_yamls repo root):
+#
+#   make kuttl_keystone
+#
+# ASSUMPTIONS:
+#
+# 1. Latest version of kuttl is installed at /usr/local/bin/kubectl-kuttl
+#    - wget https://github.com/kudobuilder/kuttl/releases/download/v0.11.1/kubectl-kuttl_0.11.1_linux_x86_64
+#    - mv kubectl-kuttl_0.11.1_linux_x86_64 /usr/local/bin/kubectl-kuttl
+#    - chmod 755 /usr/local/bin/kubectl-kuttl
+# 2. An OCP 4.10+ CRC cluster with Podified Operators has been deployed
```

Comment:
The errors you see in the output happen sometimes when running `make deploy_cleanup`. I'm not sure why they happen, but they are not related to the tests. There are actually two issues with the way you are running the tests:

1) When running `make octavia_kuttl` without any parameter, it will try to run the kuttl tess from the master branch of the `openstack-k8s-operators/octavia-operator`, but since there are none, it does nothing. To run them you should run
```
make octavia_kuttl OCTAVIA_REPO=https://github.com/cescgina/octavia-operator OCTAVIA_BRANCH=kuttl_tests
```
or 
```
make octavia_kuttl OCTAVIA_KUTTL_CONF=/path/to/octavia-operator/kuttl-test.yaml OCTAVIA_KUTTL_DIR=/path/to/octavia-operator/tests/kuttl/tests/
```
The first command will run the tests from this PR, the second will allow you to run the tests from your local environment, useful if you would like to make some changes there.

2) You don't need to run `make openstack`, `make openstack_deploy`, `make octavia` when running `make octavia_kuttl`, that target already runs all the dependencies needed for the tests. If you have an evironment with the operators already installed, then you can run `make octavia_kuttl_run` which will only run the tests (but you will still need to pass the variables mentioned in the previous point).

I should have posted this instructions as part of the PR, sorry about that. Let me know if it works for you with these changes.

Reply:
I tried to run the first command you mentioned under 1., but it got stuck again. So I cleaned up my CRC deployment and started from scratch and didn't run the deploymen comment from my comment above. Now the command you suggested fails:

```
merge config dir
++ find . -type f -name '*.yaml'
++ grep -v kustomization
+ yamls=./keystone_v1beta1_keystoneapi.yaml
+ for y in ${yamls[@]}
+ kustomize edit add resource ./keystone_v1beta1_keystoneapi.yaml
+ popd
~/install_yamls
oc kustomize /home/stack/install_yamls/out/openstack/keystone/cr | oc apply -f -
keystoneapi.keystone.openstack.org/keystone created
kubectl-kuttl assert -n openstack /home/stack/install_yamls/out/operator/mariadb-operator/tests/kuttl/tests/../common/assert_sample_deployment.yaml --timeout 180
I0202 14:43:58.910079  236945 request.go:655] Throttling request took 1.039910226s, request: GET:https://api.crc.testing:6443/apis/user.openshift.io/v1?timeout=32s
I0202 14:44:08.952348  236945 request.go:655] Throttling request took 1.047214803s, request: GET:https://api.crc.testing:6443/apis/rabbitmq.com/v1beta1?timeout=32s
I0202 14:44:20.004955  236945 request.go:655] Throttling request took 1.046021391s, request: GET:https://api.crc.testing:6443/apis/monitoring.coreos.com/v1?timeout=32s
I0202 14:44:30.054918  236945 request.go:655] Throttling request took 1.046085219s, request: GET:https://api.crc.testing:6443/apis/certificates.k8s.io/v1?timeout=32s
I0202 14:44:41.255844  236945 request.go:655] Throttling request took 1.043263517s, request: GET:https://api.crc.testing:6443/apis/autoscaling.openshift.io/v1?timeout=32s
I0202 14:44:51.468994  236945 request.go:655] Throttling request took 1.048665193s, request: GET:https://api.crc.testing:6443/apis/autoscaling/v2beta2?timeout=32s
I0202 14:45:02.663020  236945 request.go:655] Throttling request took 1.043128347s, request: GET:https://api.crc.testing:6443/apis/networking.k8s.io/v1?timeout=32s
I0202 14:45:12.876264  236945 request.go:655] Throttling request took 1.044985532s, request: GET:https://api.crc.testing:6443/apis/autoscaling/v2beta1?timeout=32s
assert is valid
make octavia_kuttl_run
make[1]: Entering directory '/home/stack/install_yamls'
INSTALL_YAMLS=/home/stack/install_yamls kubectl-kuttl test --config /home/stack/install_yamls/out/operator/octavia-operator/kuttl-test.yaml /home/stack/install_yamls/out/operator/octavia-operator/tests/kuttl/tests
Error: open /home/stack/install_yamls/out/operator/octavia-operator/kuttl-test.yaml: no such file or directory
make[1]: *** [Makefile:800: octavia_kuttl_run] Error 255
make[1]: Leaving directory '/home/stack/install_yamls'
make: *** [Makefile:804: octavia_kuttl] Error 2
```

Reply:
I could reproduce this issue in my machine, it seems that I missed a dependency in the `install_yamls` target. Already updated the other PR, please checkout the latests changes and try again, it should now work (or at least it works in my environment).

Reply:
Thanks. I can confirm it works now.